plugins/inputs/nfsclient/nfsclient_test.go

```diff
@@ -0,0 +1,167 @@
+package nfsclient
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/influxdata/telegraf/testutil"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func readMountstatsFile() string {
+	path := "./mountstats"
```

Comment:
It seems like an unwritten convention to put those files into a `testdata` directory (e.g. https://github.com/influxdata/telegraf/tree/master/plugins/inputs/snmp). Could you please move the file there?

Reply:
See 937fdcdf452f0e1de19150c63f34be62257507b0

Reply:
And e8bec755b

Reply:
Please mark as resolved.